src/TimeSeries/widget/ui/TimeSeries.css

```diff
@@ -0,0 +1,674 @@
+/* nvd3 version 1.8.4 (https://github.com/novus/nvd3) 2016-07-03 */
```

Comment:
This will most likely fails somewhere in the future where you upgrade nvd3 but forget to update this file. You should import the css file from nvd3 here, webpack can handle the import for you.